client/data/blogging-prompt/use-blogging-prompt.ts

```diff
@@ -5,34 +5,45 @@ import wp from 'calypso/lib/wp';
 export interface BloggingPrompt {
 	id: string;
 	text: string;
+	answered: boolean;
+	answered_users_count: number;
+	answered_users_sample: [ string ];
 }
 
-const selectPrompt = ( response: any ): BloggingPrompt | null => {
-	const prompt = response && response.prompts && response.prompts[ 0 ];
-	if ( ! prompt ) {
+const selectPrompts = ( response: any ): [ BloggingPrompt ] | null => {
+	const prompts = response && response.prompts;
+	if ( ! prompts ) {
 		return null;
 	}
-	return {
-		id: prompt.id,
-		text: prompt.text,
-	};
+	return prompts.map(
+		( prompt: any ): BloggingPrompt => ( {
+			id: prompt.id,
+			text: prompt.text,
+			answered: prompt.answered,
+			answered_users_count: prompt.answered_users_count,
+			answered_users_sample: prompt.answered_users_sample,
+		} )
+	);
 };
 
-export const useBloggingPrompt = ( siteId: string ): UseQueryResult< BloggingPrompt | null > => {
+export const useBloggingPrompts = (
+	siteId: string,
+	page: number
+): UseQueryResult< [ BloggingPrompt ] | null > => {
 	const today = moment().format( 'YYYY-MM-DD' );
 
 	return useQuery(
 		// Blogging prompts are the same for all sites, so can be cached only by date.
-		[ 'blogging-prompts', today ],
+		[ 'blogging-prompts', today + '-' + page + 1 ],
```

Comment:
This seems a bit hacky for pagination; I think it's worth updating the endpoint to do it properly. What do you think of adding a `page` query param that indicates which page number to return?

Reply:
Oops that was left in by mistake. The `+ 1` was a cache buster.